samples/bluetooth/unicast_audio_client/boards/nrf52840dk_nrf52840.conf

```diff
@@ -0,0 +1,15 @@
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_CENTRAL_ISO=y
+
+# For LC3 the following configs are needed
+CONFIG_FPU=y
+CONFIG_LIBLC3CODEC=y
+# The LC3 codec uses a large amount of stack. This app runs the codec in the work-queue, hence
+# inctease stack size for that thread.
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+# LC3 lib requires floating point support in the c-lib NEWLIB is one way of getting that.
+CONFIG_NEWLIB_LIBC=y
```

Comment:
I did not test LC3 can run on the nrf52 device with regards to CPU resources and RAM.

Reply:
Hmm, good point. Didn't actually test that, but can give it a try before merging this :)

Reply:
Hmm, actually I do not have a way of testing LC3 on the nRF52840 at this point, so will remove the LC3 from the board files.